chapters/hpmor-chapter-096.tex

```diff
@@ -1,9 +1,14 @@
 \partchapter{Roles}{VII}
 
-\section[The fourth meeting:\\
-(4:38\pm, April 17th, 1992)]{The fourth meeting:\\
-(4:38\pm, April 17th, 1992)\protect\footnotemark}
-\authorsnotetext{For those who have not read canon: the wooden sign has somewhat changed, but the inscription here is the same as in J.~K.~Rowling’s original.}
+\begin{chapterOpeningAuthorNote}
+For those who have not read canon: the wooden sign has somewhat changed, but the inscription here is the same as in J.~K.~Rowling’s original.
+\end{chapterOpeningAuthorNote}
+% \begin{chapterOpeningQuote}
```

Comment:
We don't need an empty commented-out opening quote, I think?

Reply:
ok, removed all the empty ones from all files